doc/src/gotchas.rst

```diff
@@ -270,6 +270,33 @@ trick for getting tab completion in the regular Python console.
     See also `What is the best way to create symbols?
     <https://github.com/sympy/sympy/wiki/Faq>`_ in the FAQ.
 
+.. _symbols:
+
+Functions
+---------
+
+A function like ``f(x)`` can be created by defining the Function
+and the variable:
+
+    >>> from sympy import Function
+    >>> f = Function('f')
+    >>> x = Symbol('x')
+    >>> f(x)
+    f(x)
+
+If you assign ``f(x)`` to a Python variable `f` you will lose your
+ability to copy and paste that function or to create a function
+with a different argument: ``Function('f')`` is callable, but
+``Function('f')(x)`` is not:
+
+    >>> f = f(x)
+    >>> f
+    f(x)
+    >>> f(x)
```

Comment:
Perhaps a better demonstration would use a different argument e.g. `f(1)`.

Reply:
```suggestion
    >>> f(1)
```
Is the above okay?

Reply:
Yes but the idea would be to show how it works in both cases.

Reply:
The normal case is shown in L284 and L285. Should I add a normal call of `Function('f')(1)` here too?

Reply:
The point is to show that if
```
f1 = Function('f')
f2 = Function('f')(x)
```
then you can call `f1` with different arguments e.g. `f1(1)` but that doesn't work for `f2(1)`. It is possible though with subs e.g. `f2.subs(x, 1)`.

Reply:
I have changed the example.